handbook/engineering/cloud/index.md

```diff
@@ -122,13 +122,14 @@ By default, members of the team will provide a brief update about the goals they
   - [Alan Harris](../../../company/team/index.md#alan-harris-he-him)
   - [Artem Ruts](../../../company/team/index.md#artem-ruts-he-him)
   - [Alex Russell-Saw](../../../company/team/index.md#alex-russell-saw-he-him)
+  - I. G. estimated start date is February 1st 2021
 
 ## Growth plan
 
-_Updated 2020-11-18_
+_Updated 2020-12-07_
 
-We are looking to grow this team to 8 engineers by hiring an additional [Frontend Engineer](../hiring/software-engineer-frontend.md) and a [Backend Engineer](../hiring/software-engineer-backend.md).
+We are looking to grow this team to 8 engineers by hiring an additional [Frontend Engineer](../hiring/software-engineer-frontend.md).
```

Comment:
Team is already at 8 engineers (with you being an IC) so what is the plan here? Are we going to grow to 9 or are we pausing for now?

Reply:
We need another frontend engineer so that @arussellsaw can focus on his core competencies. I think the extra teammate would be OK from a management perspective.